doc/ssl/SSL_SESSION_get_time.pod

```diff
@@ -52,6 +59,8 @@ valid values.
 
 SSL_SESSION_set_time() and SSL_SESSION_set_timeout() return 1 on success.
 
+SSL_SESSION_get0_hostname returns the current value.
```

Comment:
This provides no information at all! It needs to be more specific about what value we are talking about. Also can this ever fail or return NULL?